WooCommerce/Classes/ViewRelated/Products/SKU Scanner/ProductSKUInputScannerViewController.swift

```diff
@@ -59,8 +59,9 @@ private extension ProductSKUInputScannerViewController {
 
 private extension ProductSKUInputScannerViewController {
     enum Localization {
-        static let title = NSLocalizedString("Scan barcode to update SKU", comment: "Navigation bar title for scanning a barcode to use as a product's SKU.")
-        static let instructionText = NSLocalizedString("Scan product barcode",
+        static let title = NSLocalizedString("Scan barcode or QRCode to update SKU",
```

Comment:
I think we should use "QR code" instead of the "QRCode" as a single word. This comment applied to other strings as well. 

```suggestion
        static let title = NSLocalizedString("Scan barcode or QR code to update SKU",
```

Reply:
Done a2478aa8d5354b41a1ce2199941eb6c4bffa53dc